DEVDOCS-2455: Removed Store Design reference (#908)
bigcommerce/dev-docs: docs/stencil-docs/storefront-customization/using-custom-fonts-and-icons.md

### Google Fonts

> [Google Fonts](https://fonts.google.com/) are a collection of open source fonts available for use. The base Cornerstone theme uses Karla and Montserrat. Google Fonts come included in the Cornerstone theme.  

Reviewer: [Google Fonts](https://fonts.google.com/) are a collection of open-source fonts available for use. The base Cornerstone theme uses Karla and Montserrat. Google Fonts come included in the Cornerstone theme.

<!-- theme: {{callout_type}} -->

### CDN Handlebars
> The CDN custom Handlebars helper assumes WebDAV's default `/content/` folder, so there is no need to specify that folder.
> Fonts staged via WebDAV, as in the above example, will not show up in the local version of your theme.

</div>
</div>
</div>

> Locations where the font is not overridden in CSS will fall back on the font in `config.json`. Make sure to update the font there to prevent fallbacks.

Reviewer 2: ```suggestion
The location font will reference the settings in the `config.json` file if the CSS does not override the font in a specific location.
```
This might make the message clearer. Let me know if I'm obscuring the meaning by changing the wording, @ektravel .

Author: @uwikaiddi What about this?
Files that do not contain font-overriding CSS code will use the CSS settings in the `config.json` file.

Reviewer 2: @ektravel, awesome!

`npm install`

### Calling theme icons

A Stencil theme's icons are delivered via a single SVG sprite, which is embedded on the page in
`<theme-name>templates/layout/base.html`. This sprite is generated via the Grunt task `grunt svgstore`.

The task takes individual SVG files for each icon (in the theme's `assets/icons/` subdirectory) and bundles them together, to be inlined on the top of the theme, inside a Handlebars partial.

> You can then call each icon in a similar way to an inline image, via the following HTML:

Reviewer: You can then call each icon in a similar way to an inline image via the following HTML:
